docs/sources/metainfo/service.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN" "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd" [
+<!ENTITY % BOOK_ENTITIES SYSTEM "AppStream.ent">
+%BOOK_ENTITIES;
+]>
+
+<section id="sect-Metadata-Service">
+	<title>Services</title>
+
+	<section id="spec-service-introduction">
+		<title>Introduction</title>
+		<para>
+                        A service component is any software that is started and supervised by
+                        the Operating Systems "init" facility, such as systemd.
+		</para>
+		<para>
+                        A service component is any software that is started and supervised by
+                        the Operating Systems "init" facility, such as systemd.
+		</para>
+		<para>
+			The file described in this document is built upon the generic component metadata with fields specific for services (see <xref linkend="sect-Metadata-GenericComponent"/>).
```

Comment:
Don't say "file" as AppStream metadata might not be retrieved as file, and this generally is a bit confusing.
Usually, we use "component" here. Additionally, I try to avoid using the term "document" to refer to the text people are reading, because since there are XML documents and YAML documents as well as two types of AppStream "documents" this is confusing.
So, I'd probably have written something like "The component described here is built upon...."